### PR TITLE
Add a flag to auto-configure Gitlab

### PR DESCRIPTION
**Pull request by angeliski**
That PR is to update the Gitlab to include autogeneration option, like Travis and Jenkins.

The motivation is the issue in jhipster/jhipster-online#80

-   Please make sure the below checklist is followed for Pull Requests.

-   [x] [Travis tests](https://travis-ci.org/jhipster/generator-jhipster/pull_requests) are green
-   [X] Tests are added where necessary
-   [X] Documentation is added/updated where necessary
-   [X] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

File: generators/ci-cd/index.js [hunk 1]
```diff
@@ -46,6 +46,13 @@ module.exports = class extends BaseGenerator {
             defaults: false,
             description: 'Automatically configure Jenkins'
         });
+
+        // Automatically configure Jenkins
```
Comment (atomfrede) [inline generators/ci-cd/index.js]: Let's remove confusion and correct the comment😁

Comment (angeliski) [reply generators/ci-cd/index.js]: Done!

**Comment by pascalgrimaud** [comment]
thanks and congrats for your 1st contribution @angeliski !

**Comment by jdubois** [comment]
Awesome, congrats for your 1st contribution @angeliski !!

**Comment by angeliski** [comment]
Thanks @pascalgrimaud and @jdubois More in coming :)

**Pull request merged**